src/main/java/org/spdx/maven/CreateSpdxMojo.java

```diff
@@ -478,9 +478,16 @@ public class CreateSpdxMojo extends AbstractMojo
      * If true, use ${project.groupId}:${artifactId} as the SPDX package name.
      * Otherwise, ${project.name} will be used
      */
-    @Parameter
+    @Parameter( property = "spdx.useArtifactID" )
     private boolean useArtifactID;
 
+    /**
+     * If true, adds an external reference to every package with category "PACKAGE-MANAGER", type "purl"
+     * and locator "pkg:maven/${project.groupId}/${project.artifactId}@${project.version}".
+     */
+    @Parameter( property = "spdx.generatePurls" )
```

Comment:
Thanks to you for the quick review!

Good point here. I've URL encoded the `artifactId`, which replaces the whitespaces with `+`. This will also encode any other symbol not valid in a PURL.

Reply:
Agree, it can be very handy. Set default value to `true` in ae2294fe80f53786cca1106f201a29afe51c7027.